import/index_java/src/org/vufind/index/RelatorContainer.java

```diff
@@ -50,6 +58,24 @@ public Set<String> getKnownRelators()
         return knownRelators;
     }
 
+    public Set<String> getRelatorPrefixesToStrip()
+    {
+        // Populate set if empty:
+        if (relatorPrefixesToStrip.isEmpty()) {
```

Comment:
If somebody doesn't use these values and resets the config to an empty list, getConfigSection() will be called over and over again, which might produce lots of warnings. Woulnd't it be better to use null as default for the class variable and test against that?

Reply:
Thanks, @mtrojan-ub, that's a very good point. This code was modeled on PunctuationContainer.java, which contains the same issue. I've opened #2597 to fix that, and closer examination of the code here made me realize that this could be done more simply, so I've further simplified the logic. I also removed a `System.out.println` debug message that I had not intended to commit.